### DISCUSSION TITLE
major-updates-to-netescapades-aspnetcore-security-headers/

### DISCUSSION THREAD

**Discussion by giscus** [andrewlock/blog-comments#435] [Announcements]
# major-updates-to-netescapades-aspnetcore-security-headers/

In this post I describe the recent major changes to NetEscapades.AspNetCore.SecurityHeaders, a NuGet package for adding security headers to your apps.

https://andrewlock.net/major-updates-to-netescapades-aspnetcore-security-headers/

**Comment by ariellourenco** [comment]
Hi @andrewlock, 

First of all, thanks for take the time to put together all these security headers in an easy-to-use library. This is more like a question than a feedback but while I was reading your blog post the `Strict-Transport-Security: max-age=31536000; includeSubDomains` got my attention. I wondering isn't this header doing the same thing that [HSTS middleware](https://learn.microsoft.com/en-us/aspnet/core/security/enforcing-ssl) does?

**Reply by andrewlock** [reply]
Yep, it's the same thing essentially
